src/script/conversation/ConversationRepository.ts

```diff
@@ -396,7 +397,7 @@ export class ConversationRepository {
      * we need to add this creator_client to conversation creation payload
      * for creating MLS conversations
      */
-    if (this.teamState.isMLSEnabled()) {
```

Comment:
do we still need to make sure MLS is enabled in the team features?

Reply:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/63250054/178016506-c8b230a6-afd3-4efa-92d0-8415a3c60e7d.png">
this is already changed